cg_openmm/parameters/secondary_structure.py

```diff
@@ -49,19 +50,25 @@ def get_native_contacts(cgmodel, native_structure, native_contact_distance_cutof
     return native_contact_list, native_contact_distances
 
 
-def expectations_fraction_contacts(list_native_contacts, distance_cutoff, temperature_list, frame_begin=0, output_directory="output", output_data="output.nc", num_intermediate_states=0):
+def expectations_fraction_contacts(native_contact_list, native_contact_distances, temperature_list, pdb_file_list, native_contact_cutoff_ratio=1.00, frame_begin=0, output_directory="output", output_data="output.nc", num_intermediate_states=0):
```

Comment:
I'm thinking about ways this could be more easily be modularized. 

Would it make sense to separate out the functionality of taking the list and seeing which of them are within the proper distance?  It seems like the "fraction contact" list could be an arbitrary set of contacts (determined however one wants)

In theory, one could separate out the structural expectation code completely from the reweighting - all reweighting needs is a structural observable assigned for each structure that has an energy assigned that will be passed to MBAR.  So the "determine what Q is" can be separated out entirely from the reweighting part.  That would make it easier to reuse the code for any other structural observable. 

The code looks correct; I would just do the manual regression test of taking the same trajectory and running it with both old and new code to make sure the answers are the same to within rounding (ideally higher precision that PDB rounding, as we discussed in another issue)